pkg/network/namescheme/networknamescheme.go

```diff
@@ -138,6 +138,16 @@ func PodHasOrdinalInterfaceName(podNetworkStatus map[string]networkv1.NetworkSta
 	return false
 }
 
+// PodHasOrdinalInterfaceName2 checks if the given pod network status has at least one pod interface with ordinal name
+func PodHasOrdinalInterfaceName2(networkStatuses []networkv1.NetworkStatus) bool {
```

Comment:
This issue was raised twice, please see:
- https://github.com/kubevirt/kubevirt/pull/13018#discussion_r1793291246
- https://github.com/kubevirt/kubevirt/pull/13018#discussion_r1799014788

It will be fixed in a follow-up.